### PR TITLE
Replaced deprecated Pimcore\Cache\Runtime with Pimcore\Cache\RuntimeCache

### LINKED ISSUE #301: Do not use deprecated methods from Pimcore\Cache\Runtime
### Improvement description

Related to https://github.com/pimcore/pimcore/pull/12548

### PR DESCRIPTION
**Pull request by aashan10**
Resolves #301

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "pear/archive_tar": "^1.4.3",
     "pimcore/number-sequence-generator": "^1.0.5",
     "pimcore/object-merger": "^2.4.5 || ^3.0",
-    "pimcore/pimcore": "^10.4",
+    "pimcore/pimcore": "^10.4.5",
```
Comment (dvesh3) [inline composer.json]:
```suggestion
    "pimcore/pimcore": "^10.5",
```

**Comment by blankse** [comment]
@aashan10 @dvesh3 I created a PR for bumping pimcore version to 10.5. We need to drop Pimcore 6.9 tests then.
https://github.com/pimcore/customer-data-framework/pull/308

**Comment by dvesh3** [comment]
@aashan10 please rebase on 3.x branch to fix the tests.

**Comment by aashan10** [comment]
@dvesh3 done 👍🏽

**Review by dvesh3 — APPROVED** [review]

**Pull request merged**